lib/ui/Editor.js

```diff
@@ -621,6 +627,16 @@ Editor.prototype._updateCursor = function () {
     self.screen.program.hideCursor();
   }
 };
+Editor.prototype._clipCursorToContent = function (cursor) {
+  var self = this;
+  var linesCount = self.textBuf.getLastRow();
+  var lineLength = self.textBuf.lineLengthForRow(cursor.y);
```

Comment:
This might not be right after [this `max`/`min`](https://github.com/mjurczyk/slap/blob/feature-cli-at-line/lib/ui/Editor.js#L635).

Are you sure this function is necessary in the first place? i.e. doesn't `setHeadPosition` automatically clip?